### DISCUSSION TITLE
Issues with accessing Kibana default T-POT dashboard a few days after installation?

### DISCUSSION THREAD

**Discussion by mitcabbo** [telekom-security/tpotce#814] [Q&A]
Hi everyone, I am new here and decided to use T-POT to analyze attack information for a cybersecurity final project. A few days after initially installing T-POT on an Azure VM, I am seeing an Elastic "Create your first dashboard" screen instead of the previous T-POT default dashboard. Any ideas on what went wrong? I don't want to lose all the hundreds of thousands of data entries that were collected over the last few days. Thanks in advance!

**Comment by rootdump** [comment]
I am having the same issues. Have had tpot running for a few months and all working fine. I am now getting create your first dashboard and as yet have found no way of going back. Have you managed to find a way?

**Reply by mitcabbo** [reply]
I tried restarting T-POT using "systemctl restart tpot" - no luck.
I tried restarting the whole VM - no luck.
I checked my network settings to make sure my device was able to connect to the T-POT web UI - no luck.

I recently reached out to Elastic to see if they had any similar issues reported to them, seeing as their information was the only thing I could find on the Kibana dashboard page - no luck.

**Reply by rootdump** [reply]
Thanks for the reply. I am not sure what has happened. I have not changed any code and has been running perfect since the install. I don't really want to re-build the system but there may be no other option. I will try and fix and if I do I well let you know.

**Reply by mitcabbo** [reply]
We implemented this system for our final project due in three days and I am now left with nothing. I hope to hear back from someone soon. Good luck!

**Comment by rootdump** [comment]
Have fixed it. You need to download kibana.export.ndjson from https://github.com/telekom-security/tpotce/wiki/Import---Export-Kibana-Objects

Then go to kibana > stack management > Saved Objects > Import *Then import that file.

Mines now working.

**Reply by mitcabbo** [reply]
Mine is also now working. What the heck went wrong??

**Reply by Excannor** [reply]
Is there any possibility of making this file available again? I am having the same problem.

**Comment by DevmanusHere** [comment]
![image](https://github.com/telekom-security/tpotce/assets/116114483/838e766c-f76d-47ba-ba7e-bf13046b63ab)
I tried systemctl restart tpot but i cannot make them up 
please help

**Reply by cybraman** [reply]
same issue happening with me unable keep running up the kibana and elasticsearch
